.github/workflows/lint-yml.yml

```diff
@@ -0,0 +1,22 @@
+name: lint-yml
+
+on:
+  push:
+    branches: ["main"]
+    paths:
+      - "**.yml"
+  pull_request:
+    paths:
+      - "**.yml"
+
+jobs:
+  lint-yml:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get yamlfmt
+        run: |
+          LATEST_VERSION=$(curl -s https://api.github.com/repos/google/yamlfmt/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | sed 's/^v//')
+          ./.github/scripts/get-yamlfmt.sh "Linux" "x86_64" "$LATEST_VERSION"
```

Comment:
You could have used

`go install github.com/google/yamlfmt/cmd/yamlfmt@latest`

It would have been simpler

Reply:
I looked for yamlfmt action and it's very disappointing

https://github.com/yk-lab/yamlfmt-action

Or this disappointing one

https://github.com/credfeto/action-yaml-format

The reformat is a sed 😂


I found this one, but yes it use setup-go then install it
https://github.com/norwd/fmtya

Reply:
Haha, yeah, it's slim pickings for this tool.

I did find https://github.com/supplypike/setup-bin which does work for simply downloading the binary files and putting the in `$PATH`, but then I lose checksum and signing (if applicable) validation.

Reply:
[And https://github.com/jaxxstorm/action-install-gh-release](https://github.com/jaxxstorm/action-install-gh-release) works well too

Reply:
I guess there is no silver bullet for this use case as the validation steps can be quite varied across repositories.